web-client/jest-integration.config.ts

```diff
@@ -15,15 +15,18 @@ const config: Config = {
   //   '**/web-client/integration-tests/**/?(*.)+(spec|test).[jt]s?(x)',
   //   '**/web-client/integration-tests-public/**/?(*.)+(spec|test).[jt]s?(x)',
   // ], // Uncomment testMatch to run all integration tests in integration-tests + integration-tests-public
-  moduleNameMapper: pathsToModuleNameMapper(tsconfig.compilerOptions.paths, {
+  moduleNameMapper: {
+    ...pathsToModuleNameMapper(tsconfig.compilerOptions.paths, {
     prefix: '<rootDir>/../',
   }),
+    '^broadcast-channel$': '<rootDir>/jest.mock-broadcast-channel.ts',
```

Comment:
There is probably a more elegant way to ensure that we don't have an open broadcast channel after tests finish running, but this allows us to update jest. Plan on coming back to this